### ISSUE TITLE
Functionality python comparison operators changes upon import keras2onnx

### ISSUE THREAD

**Issue by adder** [onnx/keras-onnx#587]
Hey,
As demonstraded in below code example, when loading the keras2onnx module, the functionality of the python comparison operators changes.
Is this expected behaviour?

```
import tensorflow as tf
zs = tf.zeros([2,2])
print(zs == 0)
import keras2onnx
print(zs == 0)

```

```
: tf.Tensor(
: [[ True  True]
:  [ True  True]], shape=(2, 2), dtype=bool)

: False
```

This is done with tensorflow 2.2 and keras2onnx 1.7.0 and onnx-tf   1.6.0 (I'm stuck to these versions)

**Comment by wenbingl** [comment]
Yes, Keras converter needs work with tensorflow.compat.v1.disable_tensor_equality(). 
you can re-enable it if it is bothering you.

**Comment by adder** [comment]
I worked around the problem by importing the keras2onnx module inside the function where I use keras2onnx.
When imported directly in the global space, it was interfering with the rest ofmy tensorflow code.
